src/waveform/renderers/waveformrendermark.cpp

```diff
@@ -75,17 +75,16 @@ void WaveformRenderMark::draw(QPainter* painter, QPaintEvent* /*event*/) {
                     if (sampleEndPosition != Cue::kNoPosition) {
                         DEBUG_ASSERT(samplePosition < sampleEndPosition);
                         double currentMarkEndPoint =
-                                m_waveformRenderer
-                                        ->transformSamplePositionInRendererWorld(
-                                                sampleEndPosition);
+                                m_waveformRenderer->transformSamplePositionInRendererWorld(
+                                        sampleEndPosition);
+                        QColor color = pMark->fillColor();
+                        color.setAlphaF(0.5);
                         painter->fillRect(
-                                QRect(QPoint(currentMarkPoint,
-                                              m_waveformRenderer->getHeight() -
-                                                      5),
+                                QRect(QPoint(currentMarkPoint, 0),
                                         QPoint(currentMarkEndPoint,
                                                 m_waveformRenderer
                                                         ->getHeight())),
-                                pMark->fillColor());
+                                QBrush(color, Qt::BDiagPattern));
```

Comment:
```suggestion
                                QBrush(color, Qt::Dense7Pattern));
```
I find the illusion created by the BDiagPattern scrolling with waveform movement quite distracting. How about Dense7Pattern?

Reply:
Yes, I noticed that as well. I think it might be fixed with thicker lines, but I didn't find a way to do that. I need to rewrite it anyway to use waveformmarkrange instead of waveform ark, because apparently it disappears when the cue mark leaves the left edge of the screen.